vue-frontend/src/components/ios-app-development/LandingSection.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <div
+    class="tw-bg-gradient-to-t tw-from-[#F77D8D]/[31.07%] md:tw-from-[#F77D8D]/[27.59%] tw-to-[#FFFFFF] tw-pb-12"
+  >
+    <div class="tw-container tw-p-4 tw-text-left md:tw-flex xl:tw-my-8">
+      <div class="md:tw-flex-1 lg:tw-mt-16">
+        <h1
+          class="tracking-[0.08rem] tw-pr-2 tw-mb-2 sm:tw-mb-0 tw-font-inter-semibold md:tw-font-inter-bold md:tw-text-[2.75rem] xl:tw-text-[3.75rem] tw-text-[2rem] md:tw-leading-[3.625rem] xl:tw-leading-[5.625rem] tw-leading-[2.6rem] tw-text-black-core/[0.87] md:tw-font-inter-bold"
+        >
+          iOS App Development Company
+        </h1>
+        <p
+          class="tw-font-inter-medium md:tw-font-inter-regular tw-pt-2 xl:tw-text-[1.25rem] tw-text tw-text-[1rem] xl:tw-leading-[1.875rem] 2xl:tw-w-4/5 tw-leading-[1.5rem] tw-text-black-core/[0.87]"
+        >
+          We craft tailor-made iOS apps that redefine user experience and put
+          you ahead of the game. Our team work with passion and precision to
+          transform your business vision into reality, giving your brand the
+          edge it needs in today's competitive marketplace.
+        </p>
+
+        <router-link
+          class="tw-relative tw-z-10 tw-ml-0 xl:tw-w-[290px] tw-hidden md:tw-block tw-my-8 lg:tw-mt-8 lg:tw-mb-4 xl:tw-my-8 gradient-btn consultation-btn tw-h-[3.3rem] tw-w-max tw-items-left tw-rounded-full"
+          to="/contact"
+          @click.native="mixpanel.track('tap_ios_development_cta')"
+        >
+          <span
+            class="tw-font-inter-semibold tw-text-[1rem] tw-font-normal tw-leading-[1.21rem] !tw-tracking-[0] md:tw-text-[1.09375rem] md:tw-text-[1.1875rem] md:tw-leading-[1.3125rem] md:tw-leading-[1.4375rem] lg:tw-text-[1.1875rem] lg:tw-leading-[1.4375rem]"
+            >Consult Our iOS Team</span
+          >
+        </router-link>
+      </div>
+      <div
+        class="tw-flex tw-flex-col md:tw-flex-row tw-items-center tw-justify-center md:tw-flex-1"
+      >
+        <img
+          :src="landing400"
+          :srcset="`${landing400} 400w,${landing800} 800w`"
+          class="tw-mt-7 tw-overflow-visible md:tw-mt-0 tw-h-[21.49113rem] xs:tw-h-[24.49113rem] md:tw-hidden tw-w-[23.77063rem] tw-object-cover"
```

Comment:
In some devices this image line is not touching in center of button

Reply:
Its aligned.Try to refresh and check.